strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/ProxyConfigurationController.java

```diff
@@ -0,0 +1,114 @@
+package org.carlspring.strongbox.controllers.configuration;
+
+import org.carlspring.strongbox.configuration.ProxyConfiguration;
+import org.carlspring.strongbox.controllers.BaseController;
+import org.carlspring.strongbox.services.ConfigurationManagementService;
+import org.carlspring.strongbox.services.support.ConfigurationException;
+
+import io.swagger.annotations.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+/**
+ * @author Pablo Tirado
+ */
+@Controller
+@RequestMapping("/api/configuration/strongbox/proxy-configuration")
+@Api(value = "/api/configuration/strongbox/proxy-configuration")
+public class ProxyConfigurationController
+        extends BaseController
+{
+
+    private static final Logger logger = LoggerFactory.getLogger(ProxyConfigurationController.class);
+
+    private final ConfigurationManagementService configurationManagementService;
```

Comment:
I think there could be a common parent for these contollers called `BaseConfigurationController` that has this field along with:
* `getConfiguration()`
* `getConfigurationManagementService()`

...so that this sort of code doesn't need to be copied across all these controllers.